platform/configure/config.mdx

```diff
@@ -79,79 +83,83 @@ helm upgrade loft vcluster-platform --install \
+<!-- vale off -->
+### Custom HTTP headers
+<!-- vale on -->
+The platform allows you to add custom HTTP headers to all requests that are sent to the platform API. This can be useful if you want to add custom headers to all requests, such as `X-Frame-Options` or `X-XSS-Protection`.
 
 To add custom HTTP headers, use the following configuration:
 
-```yaml
+```yaml title="Custom HTTP headers configuration"
 config:
   auth:
     customHttpHeaders:
       X-Frame-Options: SAMEORIGIN
       X-XSS-Protection: 1; mode=block
 ```
 
-### Setting additional certificate authority for Platform
-If you want to have **all** your virtual clusters managed by the vCluster Platform to use a specific certificate authority to establish the connection between the virtual cluster and the platform, you can set it as a configuration option on the vCluster Platform instead of configuring it on each individual virtual cluster.
-This will enable secure communication between your Virtual Clusters and the vCluster Platform.
-As a pre-requisite, you have to [configure TLS for the vCluster Platform](/platform/configure/domain#configure-tls) first.
-```yaml
+### Setting additional certificate authority for platform
+
+If you want to have **all** your virtual clusters managed by the platform use a
+specific certificate authority to establish the connection between the virtual
+cluster and the platform, you can set it as a configuration option on the
+platform instead of configuring it on each individual virtual cluster. This
+enables secure communication between your virtual clusters and the platform.
+
+As a prerequisite, you have to [configure TLS for the platform](/platform/configure/domain#configure-tls) first.
+
+```yaml title="Additional certificate authority configuration"
 additionalCA: "(base64 encoded CA)"
 ```
 
-Also, if you need to import externally managed vCluster to the vCluster Platform, you can also enable a secure connection by grabbing the value from the existing vCluster Platform, you can do this by running:
+If you need to import externally managed virtual clusters to the platform, you can also enable a secure connection by grabbing the value from the existing platform. You can do this by running:
 
-```shell
+```bash title="Get additionalCA value"
 helm get values --namespace [namespace where vCluster Platform is deployed] [loft helm release name] --all | grep additionalCA
 ```
 
-Using the `additionalCA` value from the above command, import the vCluster using the following vCluster CLI command:
-```shell
-  vcluster platform add vcluster <VCLUSTER_NAME> \
+Using the `additionalCA` value from the preceding command, import the vCluster using the following vCluster CLI command:
+
+```bash title="Import vCluster with additionalCA"
+vcluster platform add vcluster <VCLUSTER_NAME> \
```

Comment:
Though this is a different command, we should update the command styling for the variables to match how we've been doing the other places. 

This command is outdated and should be updated to match the install docs in variables and styling:

https://www.vcluster.com/docs/platform/install/helm#installing-the-platform-helm-chart